CardsOfOpprotunity/CardsOfOpprotunity/Views/BoardGame/GameViewModel.swift

```diff
@@ -76,10 +97,10 @@ class GameViewModel: ObservableObject {
         
         if playerOneScore > playerTwo {
```

Comment:
Hint: Use this conditional statement to determine who won the game and when to present your modal view with the details of who won, along with the bet's text value.